src/core/ext/transport/chttp2/transport/internal.h

```diff
@@ -584,6 +584,7 @@ struct grpc_chttp2_stream {
 
   grpc_slice_buffer flow_controlled_buffer;
 
+  grpc_closure_list run_after_write;
```

Comment:
I'm guessing this is memset to zero and hence does not need to be inited

Reply:
I copied the pattern of the old per-transport ```run_after_write```, and that one is not initialized either.